Lesson 2 HW Ernest.playground/Contents.swift

```diff
@@ -0,0 +1,49 @@
+import Darwin
+let number1 = 2.5
+let number2 = 9.7
+let number3 = 6.9
+let number4 = 8.2
+let wholeNumber1 = Int(floor(number1))
+let fractionalNumber1 = number1.truncatingRemainder(dividingBy: 1)
+let wholeNumber2 = Int(floor(number2))
+let fractionalNumber2 = number2.truncatingRemainder(dividingBy: 1)
+let wholeNumber3 = Int(floor(number3))
+let fractionalNumber3 = number3.truncatingRemainder(dividingBy: 1)
+let wholeNumber4 = Int(floor(number4))
+let fractionalNumber4 = number4.truncatingRemainder(dividingBy: 1)
+
+let sumWholeNumbers = wholeNumber1 + wholeNumber2 + wholeNumber3 + wholeNumber4
+let sumFractionalNumbers = fractionalNumber1 + fractionalNumber2 + fractionalNumber3 + fractionalNumber4
+
+let checkParity1 = Int(wholeNumber1) % 2
+let checkParity2 = Int(wholeNumber2) % 2
+let checkParity3 = Int(wholeNumber3) % 2
+let checkParity4 = Int(wholeNumber4) % 2
```

Comment:
As an improvement, I suggest we start using correct property naming from the start.

We use digits in the property names rarely. If we want to use the numbers in the naming, we write them with words. For example, `firstNumber` instead of `number1`.